source/dataman/DataMan.cpp

```diff
@@ -61,7 +61,10 @@ void DataMan::add_stream(json p_jmsg)
         man->init(p_jmsg);
         this->add_next(method, man);
     }
-    add_man_to_path("zfp", method);
+    if (p_jmsg["compress_method"] != nullptr)
```

Comment:
I think .is_string() is the way that the library encourage people to use. It returns false if that object is nullptr, or that object is not a string. In either case it is safe to use.